packages/web/components/Player/PlayerGuild.tsx

```diff
@@ -1,78 +1,56 @@
 import { Link } from '@metafam/ds';
 import React from 'react';
 
-import polygonImage from '../../assets/chains/polygon.png';
-import xDaiImage from '../../assets/chains/xDai.png';
-import ethereumImage from '../../assets/moloch/ethereum.png';
-import hausdaoImage from '../../assets/moloch/hausdao.png';
-import metacartelImage from '../../assets/moloch/metacartel.png';
-import metaclanImage from '../../assets/moloch/metaclan.png';
-import raidGuildImage from '../../assets/moloch/raid_guild.png';
-
 type LinkGuildProps = {
   daoUrl: string | null;
   guildname: string | undefined | null;
 };
 
-export const getMolochImage = (title: string) => {
-  title = title.toLowerCase(); // eslint-disable-line no-param-reassign
-  if (title.includes('hausdao')) return hausdaoImage;
-  if (title.includes('metacartel')) return metacartelImage;
-  if (title.includes('metaclan')) return metaclanImage;
-  if (title.includes('raid guild')) return raidGuildImage;
-  if (title.includes('xdai')) return xDaiImage;
-  if (title.includes('polygon')) return polygonImage;
-  return ethereumImage;
-};
-
-const getHexChainId = (chain?: string): string => {
-  switch (chain?.toLowerCase()) {
-    case 'xdai':
-      return '0x64';
-    case 'polygon':
-      return '0x89';
-    case 'ethereum':
-    default:
-      return '0x1';
-  }
-};
-
-export const getDaoLink = (chain?: string, address?: string): string | null => {
-  if (address && chain) {
-    const hexChainId = getHexChainId(chain);
-    address = address.toLowerCase(); // eslint-disable-line no-param-reassign
-    return `https://app.daohaus.club/dao/${hexChainId}/${address}`;
-  }
-  return null;
-};
-
 export const LinkGuild: React.FC<LinkGuildProps> = ({
   daoUrl,
   guildname,
   children,
 }) => {
   if (guildname != null) {
-    return (
-      <Link
-        role="group"
-        _hover={{ textDecoration: 'none' }}
-        href={`/guild/${guildname}`}
-      >
-        {children}
-      </Link>
-    );
+    return <InternalGuildLink guildName={guildname} children={children} />;
   }
   if (daoUrl != null) {
-    return (
-      <Link
-        role="group"
-        _hover={{ textDecoration: 'none' }}
-        href={daoUrl}
-        isExternal
-      >
-        {children}
-      </Link>
-    );
+    return <DaoHausLink daoUrl={daoUrl} children={children} />;
   }
   return <>{children}</>;
 };
+
+type InternalGuildLinkProps = {
+  guildName: string;
+};
+
+export const InternalGuildLink: React.FC<InternalGuildLinkProps> = ({
+  guildName,
+  children,
+}) => (
+  <Link
+    role="group"
```

Comment:
Why is there a role=group here ? I think its creating some issues.

Reply:
![CPT2202191414-472x522](https://user-images.githubusercontent.com/10012209/154793771-87dde509-191f-4574-ac9a-570b4dea21e5.gif)

Reply:
Well, all I did was split these components into three discrete components without changing their functionality. So `role="group"` has been there all along.

Reply:
oh right. still i think it shouldn't be there

Reply:
_groupHover is used in PlayerGuildMemberships as one of the `children`

Reply:
Are you seeing those UI bugs on hover or its misbehaving only on my side?

Reply:
I see it on Chrome but not on FF